src/index.js

```diff
@@ -1,3 +1,33 @@
-import _ from "lodash";
-import "./style.css"
-console.log("Hello world")
\ No newline at end of file
+/* eslint-disable no-unused-vars */
+import _ from 'lodash'; // Unused vars disabled for lodash
+import './style.css';
+import Task from './taskClass.js';
+
+const dataBase = [new Task('Description test1', 0), new Task('Description test2', 1), new Task('Description test3', 2)];
```

Comment:
- [ ] It would be better if you add the `completed: [bool]`  to the objects in the array. i,g `new Task('Description test1', 0, false)`. Because you add this property to the constructor so the todo must have this property.  😄 👍

Reply:
Hello @KatIsCoding and @BrianSammit, allow me to share my thoughts on this. Since the TaskClass has the value of `completed` defaulted to `false`, there is really no need to add `completed: [bool]`. Every Task object created will have this property by default and this addition will be redundancy in code. Try and console log the `dataBase` without doing this addition to verify that the `completed` property is added to each `object` because of how it is already defined in the `Class`.